Add basic issue and PR templates (#184)
jamwaffles/embedded-graphics: embedded-graphics/README.md

There may be other drivers out there we don't know about yet. If you know of a driver to add to this list, please open [an issue](https://github.com/jamwaffles/embedded-graphics/issues/new)!

## Development setup

> Ensure you have a reasonably recent version of Rust installed, preferably through <https://rustup.rs>.

Reviewer: ```suggestion
Ensure you have the latest stable version of Rust installed, preferably through <https://rustup.rs>.
```

I would like to target the latest stable version for now, because e-g and the whole embedded ecosystem is relatively new and constantly changing. When e-g is more mature this should change to support the latest n stable releases.

### Ubuntu/Linux Mint

```bash

> # Ensure rustfmt is installed

Reviewer: If you agree with my previous comment this should also include `rustup update`.
